Makefile

```diff
@@ -19,6 +19,8 @@ VERSION_BUILD ?= 1
 RAW_VERSION=$(VERSION_MAJOR).$(VERSION_MINOR).${VERSION_BUILD}
 VERSION ?= v$(RAW_VERSION)
 
+KUBERNETES_VERSION ?= $(shell cat pkg/minikube/constants/constants.go | grep "DefaultKubernetesVersion =" | cut -d ' ' -f4 | sed 's/"//g')
```

Comment:
`egrep "^var DefaultKubernetesVersion" pkg/minikube/constants/constants.go | cut -d\" -f2 is more succinct and precise.

Reply:
done